indexer/feature_meta.cpp

```diff
@@ -1,72 +1,26 @@
 #include "indexer/feature_meta.hpp"
 
-namespace
-{
-char HexToDec(char ch)
-{
-  if (ch >= '0' && ch <= '9')
-    return ch - '0';
-  if (ch >= 'a')
-    ch -= 'a' - 'A';
-  if (ch >= 'A' && ch <= 'F')
-    return ch - 'A' + 10;
-  return -1;
-}
+#include "std/algorithm.hpp"
 
-string UriDecode(string const & sSrc)
+namespace feature
 {
-  // This code is based on
-  // http://www.codeguru.com/cpp/cpp/string/conversions/article.php/c12759
-  //
-  // Note from RFC1630: "Sequences which start with a percent
-  // sign but are not followed by two hexadecimal characters
-  // (0-9, A-F) are reserved for future extension"
-
-  string result(sSrc.length(), 0);
-  auto itResult = result.begin();
 
-  for (auto it = sSrc.begin(); it != sSrc.end(); ++it)
+string Metadata::GetWikiURL() const
+{
+  string v = this->Get(FMD_WIKIPEDIA);
+  if (!v.empty())
```

Comment:
IMHO code with smaller indents is more readable than code with wider indents. Both condition (like in this case) and it's negation (what I'm proposing) are readable, but negated condition allows you to reduce indent of the following block. Meanwhile, I'm not insisting - it's your choice.